layout/_partial/head.ejs

```diff
@@ -84,6 +84,8 @@
     <%- css("css/style.min") %>
     <%- partial("_partial/config_css") %>
     <%- js("js/jquery.min") %>
+    <%- js("js/queue") %>
```

Comment:
- Please use single quotes inside JavaScript code (change it for the entire stack of import)